testsuite/features/step_definitions/common_steps.rb

```diff
@@ -15,7 +15,12 @@
 end
 
 When(/^I mount as "([^"]+)" the ISO from "([^"]+)" in the server$/) do |name, url|
-  iso_path = "/tmp/#{name}.iso"
+  # When using a mirror it is automatically mounted at /mirror
+  iso_path = if $mirror
+               url.sub(/^http:.*\/pub/, '/mirror/pub')
+             else
+               "/tmp/#{name}.iso"
+             end
```

Comment:
Are you sure we can mount using as path a mounted file?

Reply:
According to @Bischoff mounting something inside a mount is not problem. Same goes for the root (`/`) file system Eric explained to me.